Project.toml

```diff
@@ -9,7 +9,9 @@ PrecompileTools = "aea7be01-6a6a-4083-8856-8a6e6704d82a"
 UUIDs = "cf7118a7-6976-5b1a-9a39-7adc72f591a4"
 
 [compat]
+Dates = "1.6"
 PrecompileTools = "1"
+UUIDs = "1.6"
```

Comment:
Shouldn't these all be `"1"`? If I'm reading https://discourse.julialang.org/t/psa-compat-requirements-in-the-general-registry-are-changing/104958 right

Reply:
> For now you should simply match the version of your Julia compat entry, for all your standard library compat entries.

I think our case is exactly the same as the example given in that post: since our julia compat is `julia = "1.6"`, we set the same compat (`= "1.6"`) for stdlibs

Reply:
That's a quote from the original post, but above it in the same message there is an update which suggests that we should be using `"1"`, if I'm reading it correctly.

Reply:
omg, they edited the post 🤦 

thanks for pointing that out.

My reading is that in practice `"1"` and `"1.6"` will allow that same set (when Julia compat is set to `"1.6"`)
so the difference doesn't matter in practice (we don't need to change anything), it's just simpler advice to say "put stdlib compat as `= "1"`